features/htcondor/templ/preempt.pan

```diff
@@ -0,0 +1,12 @@
+structure template features/htcondor/templ/preempt;
+
+'text' = {
+    txt = "";
+    if (CONDOR_CONFIG['wctlimit'] != 0) {
+        txt = txt + "## Configure default wallclocktime ##\n";
+        txt = txt + "PREEMPT = true\n";
+        txt = txt + "MaxJobRetirementTime = " + to_string(CONDOR_CONFIG['wctlimit']) + "\n";
```

Comment:
If you're not going to use metaconfig, then at least use `format()` for what it was designed for...

e.g.
```pan
    template = "## Configure default wallclocktime ##\n";
             + "PREEMPT = true\n"
             + "MaxJobRetirementTime = %f\n";
    txt = format(template, CONDOR_CONFIG['wctlimit']);
```